core/dbt/adapters/sql/connections.py

```diff
@@ -76,20 +78,19 @@ def add_query(
 
             cursor = connection.handle.cursor()
             cursor.execute(sql, bindings)
-
             logger.debug(
                 "SQL status: {status} in {elapsed:0.2f} seconds",
-                status=self.get_status(cursor),
+                status=self.get_response(cursor),
                 elapsed=(time.time() - pre)
             )
 
             return connection, cursor
 
     @abc.abstractclassmethod
-    def get_status(cls, cursor: Any) -> str:
+    def get_response(cls, cursor: Any) -> Union[AdapterResponse, str]:
```

Comment:
This is something that other adapter plugins may need to account for, this might cause a breaking change.

Example: [this is override in dbt-spark](https://github.com/fishtown-analytics/dbt-spark/blob/bde721eda38bdccf3f7b5c1bd0ac11b270c644d4/dbt/adapters/spark/connections.py#L276), will need to update this

Reply:
We can call this out in the release notes. I'd like to reorganize + clean up the changelog before final release, anyway

Reply:
got it! I'll add a breaking section next time I break something :) please should tap me if you want a hand cleaning the changelog up